analyses/molecular-subtyping-LGAT/README.md

```diff
@@ -20,11 +40,13 @@ The files in the `lgat-subset` were generated via `00-subset-files-for-LGAT.R` u
 bash run_subtyping.sh
 ```
 
-#### Order of scripts in analysis
+This does not run the `00-LGAT-select-pathology-dx` notebook, as that is intended to be run once and tied to a specific release (`release-v17-20200908`).
+
+#### Order of scripts in subtyping
```

Comment:
Maybe we should also add a brief description of the `00-LGAT-select-pathology-dx` notebook here, to have accounted for all of the scripts/notebooks in the module. What do you think?

Reply:
I left this out because this is a subheader under `### Run script` and the README is generally not structured like the others that have a `Folder content` section. The text right before this says 

> This does not run the `00-LGAT-select-pathology-dx` notebook

I thought it would be confusing to include it here and that the `Inclusion/exclusion criteria` section covered what was most important to know about that notebook.

Reply:
Gotcha, I think that's reasonable. 🚀